<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1149</width>
    <height>1950</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>100</horstretch>
    <verstretch>100</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>626</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="iconSize">
   <size>
    <width>60</width>
    <height>60</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>395</width>
     <height>0</height>
    </size>
   </property>
   <widget class="QLabel" name="jacketPic">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>211</width>
      <height>161</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>hello world</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="buyJacket">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>260</y>
      <width>470</width>
      <height>110</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-style:solid;
background-color:#0000FF;
color:#fff;
border-radius:20px;
}</string>
    </property>
    <property name="text">
     <string>buy</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="bankAccount">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>880</y>
      <width>470</width>
      <height>141</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="value">
     <number>1000</number>
    </property>
   </widget>
   <widget class="QPushButton" name="buyShoes">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>410</y>
      <width>470</width>
      <height>110</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-style:solid;
background-color:#0000FF;
color:#fff;
border-radius:20px;
}</string>
    </property>
    <property name="text">
     <string>buy</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buyTrousers">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>560</y>
      <width>470</width>
      <height>110</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-style:solid;
background-color:#0000FF;
color:#fff;
border-radius:20px;
}</string>
    </property>
    <property name="text">
     <string>buy</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buyCar">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>710</y>
      <width>470</width>
      <height>110</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-style:solid;
background-color:#0000FF;
color:#fff;
border-radius:20px;
}</string>
    </property>
    <property name="text">
     <string>buy</string>
    </property>
   </widget>
   <widget class="QLabel" name="jacketLabel">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>230</y>
      <width>211</width>
      <height>161</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>jacket 200€</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="shoesPic">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>380</y>
      <width>211</width>
      <height>161</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>hello world</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="trousersPic">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>540</y>
      <width>211</width>
      <height>161</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>hello world</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="carPic">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>710</y>
      <width>211</width>
      <height>161</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>hello world</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="jacketLabel_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>380</y>
      <width>211</width>
      <height>161</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>shoes 50€</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="jacketLabel_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>540</y>
      <width>331</width>
      <height>161</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>trousers 100€</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="jacketLabel_4">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>710</y>
      <width>211</width>
      <height>161</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>car 900€</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="carPic_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>870</y>
      <width>471</width>
      <height>161</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>bank account in €</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>50</y>
      <width>331</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string>Daytime clock</string>
    </property>
   </widget>
   <widget class="DigitalClock" name="clock" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>371</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="penalty">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>1050</y>
      <width>470</width>
      <height>121</height>
     </rect>
    </property>
    <property name="maximum">
     <number>9000</number>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>1420</y>
      <width>241</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>exit</string>
    </property>
   </widget>
   <widget class="QLabel" name="carPic_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>1030</y>
      <width>461</width>
      <height>161</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>penalty account in €</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="message">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>1170</y>
      <width>611</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="carPic_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>1180</y>
      <width>221</width>
      <height>161</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>message:</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="musicButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>1370</y>
      <width>720</width>
      <height>160</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
border-style:solid;
background-color:#3d3d3d;
color:#fff;
border-radius:20px;
}

QPushButton:Pressed{
border-color:#ffff00
}</string>
    </property>
    <property name="text">
     <string>Music</string>
    </property>
   </widget>
   <widget class="QLabel" name="carPic_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>1570</y>
      <width>221</width>
      <height>161</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>volume</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QDial" name="dial">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>1570</y>
      <width>271</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <zorder>buyJacket</zorder>
   <zorder>bankAccount</zorder>
   <zorder>buyShoes</zorder>
   <zorder>jacketPic</zorder>
   <zorder>buyTrousers</zorder>
   <zorder>buyCar</zorder>
   <zorder>jacketLabel</zorder>
   <zorder>shoesPic</zorder>
   <zorder>trousersPic</zorder>
   <zorder>carPic</zorder>
   <zorder>jacketLabel_2</zorder>
   <zorder>jacketLabel_3</zorder>
   <zorder>jacketLabel_4</zorder>
   <zorder>carPic_2</zorder>
   <zorder>label</zorder>
   <zorder>clock</zorder>
   <zorder>penalty</zorder>
   <zorder>radioButton</zorder>
   <zorder>carPic_3</zorder>
   <zorder>message</zorder>
   <zorder>carPic_4</zorder>
   <zorder>musicButton</zorder>
   <zorder>carPic_5</zorder>
   <zorder>dial</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DigitalClock</class>
   <extends>QWidget</extends>
   <header location="global">digitalclock.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>radioButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>612</x>
     <y>1303</y>
    </hint>
    <hint type="destinationlabel">
     <x>608</x>
     <y>1335</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
